### ISSUE TITLE
SelectedItem is always null for RadioButtonGroupview?

### ISSUE THREAD

**Issue by uksreejith** [enisn/Xamarin.Forms.InputKit#156]
Hi,

When I select a radio button the SelectedItem property of the enclosing RadioButtonGroupView is always null. But the SelectedIndex property is set correctly.

Please help. Thanks in advance.

With Regards,
Sreejith Krishnadas

**Comment by enisn** [comment]
I've just found the bug. If you define type of your property as `object` binding works but other exact types doesn't work for now until fixed.

**Comment by technololy** [comment]
true, always had this issue. i tried setting it to object, still was null.

workaround is to get the selecteditem from code behind
